src/DIRAC/Workflow/Modules/Script.py

```diff
@@ -90,20 +90,21 @@ def _executeCommand(self):
 
         remoteRunner = RemoteRunner()
         if remoteRunner.is_remote_execution():
-            outputDict = remoteRunner.execute(self.command)
+            retVal = remoteRunner.execute(self.command)
         else:
-            outputDict = systemCall(
+            retVal = systemCall(
                 timeout=0,
                 cmdSeq=shlex.split(self.command),
                 env=self.environment,
                 callbackFunction=self.callbackFunction,
                 bufferLimit=self.bufferLimit,
             )
 
-        if not outputDict["OK"]:
+        if not retVal["OK"]:
             failed = True
-            self.log.error("System call execution failed:", "\n" + str(outputDict["Message"]))
-        status, stdout, stderr = outputDict["Value"][0:3]
+            self.log.error("System call execution failed:", "\n" + str(retVal["Message"]))
+            self._exitWithError(1000)
```

Comment:
I don't see a nicer way of handling this case without major refactoring but I'm open to ideas.